llvm/lib/TargetParser/RISCVISAInfo.cpp

```diff
@@ -920,8 +920,8 @@ void RISCVISAInfo::updateImplication() {
 }
 
 static constexpr StringLiteral CombineIntoExts[] = {
-    {"zk"},    {"zkn"},  {"zks"},   {"zvkn"},  {"zvknc"},
-    {"zvkng"}, {"zvks"}, {"zvksc"}, {"zvksg"},
+    {"b"},     {"zk"},    {"zkn"},  {"zks"},   {"zvkn"},
```

Comment:
I'm concerned this will make us not interoperate with older versions of binutils that don't know about B.

Reply:
@kito-cheng WDYT?

Reply:
I removed this part.
We can support combination of B extension in a separate patch if we need it.

Reply:
Remove this from `CombineIntoExts` may cause `__riscv_b` become less useful I think?

Reply:
> Remove this from `CombineIntoExts` may cause `__riscv_b` become less useful I think?

Yeah, but I think @topperc's concern makes sense as well. Will binutils complain or just ignore unknown extensions?